### DISCUSSION TITLE
AR doesn't launch in iOS chrome browser

### DISCUSSION THREAD

**Discussion by ivoperalta** [google/model-viewer#3871] [Q&A]
(Sorry if this topic has already been opened but I didn't find that here)

Has anyone experienced a huge lag between the AR click button and its launch on chrome for iOS?
Sometimes it simply doesn't launch the AR mode. This is happening to me even on the modelviewer examples page only when using the chrome iOS browser.

**Comment by elalish** [comment]
We filed some bugs on iOS Chrome for some related things a few months ago, but I believe they've been fixed. Are you sure your iOS Chrome is up to date? And what version of iOS do you have? AR seems to be working fine on my old iPhone SE.

**Comment by ivoperalta** [comment]
Thanks and sorry for not providing this information in advance: **iOS 15.6.1, Chrome 105.5195.129.**
However, I realized that maybe it could be a matter of weight of the .usdz file because after some time of waiting the AR is in fact launched correctly.
The problem is that I have no way of indicating that the model is being loaded, so the user ends up giving up. (Any ideas are welcome)
I also noticed that this issue occurs in the same way on the model-viewer example page.

**Comment by elalish** [comment]
What device are you using? Can you post a screen recording of how slow our example page is for you? And is it significantly different than in Safari?
